setup.cfg

```
@@ -1,39 +1,40 @@
-[metadata]
+ [metadata]
 name = disease-normalizer
-version = 0.1.1
-description = VICC normalization routine for diseases
-url = https://github.com/cancervariants/disease-normalization
 author = VICC
 author_email = help@cancervariants.org
+description = VICC normalization routine for diseases
+long_description = file:README.md
+long_description_content_type = text/markdown
+home-page = https://github.com/cancervariants/disease-normalization
+license-file = LICENSE
 license = MIT
 
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+
 [options]
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+
 install_requires =
-    boto3==1.17.18
-    botocore==1.20.18; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'
-    certifi==2020.12.5
-    chardet==4.0.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-    click==7.1.2
-    fastapi==0.63.0
-    h11==0.12.0; python_version >= '3.6'
-    httptools==0.1.1
-    idna==2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-    isodate==0.6.0
-    jmespath==0.10.0; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'
-    owlready2==0.27
-    pydantic==1.8
-    pyparsing==2.4.7; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'
-    python-dateutil==2.8.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2'
-    rdflib==5.0.0
-    requests==2.25.1
-    s3transfer==0.3.4
-    six==1.15.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2'
-    starlette==0.13.6; python_version >= '3.6'
-    typing-extensions==3.7.4.3
-    urllib3==1.26.3; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'
-    uvicorn==0.13.4
-    uvloop==0.15.2
-    websockets==8.1
+    boto3
+    click
+    fastapi
+    owlready2
+    requests
+    rdflib
+    uvicorn
+
+tests_require =
+    pytest
+    pytest-cov
 
 [tool:pytest]
 addopts = --ignore setup.py --ignore alembic --doctest-modules --cov-report term-missing --cov .
```

I think we can remove the alembic part from all normalizers. I believe this was when we were using SQLAlchemy. 
```suggestion
addopts = --ignore setup.py --doctest-modules --cov-report term-missing --cov .
```